pages/onboarding/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Button, Container, Typography } from '@mui/material';
+import { NextPage } from 'next';
+import { useRouter } from 'next/router';
+
+const Index: NextPage = () => {
+  const router = useRouter();
+  const onSkip = () => {
+    router.push('/');
+  };
+
+  return (
+    <Container maxWidth="lg" className="flex justify-between pt-4">
+      <Typography variant="h1" className="text-3xl">
+        OnBoarding
+      </Typography>
+      <Box className="flex items-center gap-4">
+        <Button variant="contained" color="secondary" onClick={onSkip}>
+          Skip
+        </Button>
```

Comment:
I don't believe the click handler is necessary. Instead, we can just wrap the `Button` with a NextJS `Link` component ([docs](https://nextjs.org/docs/api-reference/next/link)).

```suggestion
        <Link href="/" passHref>
          <Button variant="contained" color="secondary">Skip</Button>
        </Link>
```

Reply:
Yea a button should do something on the page, a link takes you somewhere. So semantically it makes sense

Fixed